### DISCUSSION TITLE
有人遇到过在 A 项目里修改配置，最后发布到 B 项目里吗？

### DISCUSSION THREAD

**Discussion by wcp1231** [apolloconfig/apollo#3702] [Q&A]
我们好像遇到过很多次，在 A 项目的 namespace 里修改，最后发现 B 项目的 namespace 的配置被改了

因为被影响的是其他业务的 B 项目，所以往往都是过了几天才发现

看 Apollo 的源码，修改配置的时候最后是通过 request body 里的 itemId 查询某个配置项进行修改，并没有验证这个 ItemId 查出来的配置项是不是属于当前这个 namespace。
因此不知道是不是浏览器发起修改请求的时候 Request Body 里的 ItemId 就已经错了。或者还有什么其他可能吗？

不知道有没有人遇到过类似的情况，我在 Issues 里没有搜到类似的反馈

**Comment by nobodyiam** [comment]
可以升级下 apollo 版本看看，最近的版本已经增加了校验逻辑

https://github.com/ctripcorp/apollo/blob/dc41c934349af28631cc5b2d9b2bdf75eac9d1f3/apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/controller/ItemController.java#L91-L107

**Reply by wcp1231** [reply]
好的，我们处理后观察一下，感谢🙏
